S/Statistics/Compat.toml

```diff
@@ -0,0 +1,2 @@
+[1]
+julia = "1.9.0-1"
```

Comment:
@KristofferC I put 1.9 because `]test Statistics` fails on Julia 1.8 and earlier (due to small changes in inference or exception types).

Reply:
I think that should be fine